config/application.rb

```diff
@@ -249,5 +249,7 @@ module ::Reporting; end
     config.active_storage.variable_content_types += ["image/svg+xml"]
 
     config.exceptions_app = self.routes
+
+    config.active_storage.service_urls_expire_in = 60.minutes
```

Comment:
I guess that his applies for all ActiveStorage URLs? We should probably limit this to the reports.

Reply:
You are right. I have not thought of that.